src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Markup/XamlTypeMapper.cs

```diff
@@ -245,7 +247,7 @@ public XamlTypeMapper(string[] assemblyNames)
 
 #endregion Methods
 
-#region Properties
+        #region Properties
```

Comment:
@singhashish-wpf  seems formatting moved.

Reply:
updated in commit https://github.com/dotnet/wpf/pull/4968/commits/1e55e8532fffbff1dcb526e91fbbca6dd324c5ab